web/views/pair_retro_view.ex

```diff
@@ -1,10 +1,6 @@
 defmodule Pairmotron.PairRetroView do
   use Pairmotron.Web, :view
 
-  def value_from_changeset(changeset, value) do
-    elem(Ecto.Changeset.fetch_field(changeset, value), 1)
-  end
-
```

Comment:
Moved to ViewHelpers which is autoimported into Pairmotron.Web for views, since this is also used for groups.